config/environments/development.rb

```diff
@@ -77,6 +77,7 @@
   config.hosts << ".bops-care.link"
   config.hosts << "southwark.bops.web"
   config.hosts << "southwark.bops.localhost"
+  config.hosts << "southwark.bops-applicants.localhost"
```

Comment:
Why do we need to have `bops-applicants` here?

Reply:
Taking it out, had some issue getting servers running but will do as separate PR